chargeOclock/Source/Models/Client.swift

```diff
@@ -34,13 +34,13 @@ class Client {
 		self.isDirty = false
 	}
 
-	func toJSON() -> Data? {
-		var data: [String: Any] = ["name": name]
+	func toData() -> Data? {
```

Comment:
no, we want toJSON here, but the method "toJSON" shouldn't return a Data object, instead it should return a dictionary
the serialization should be done at a lower level